Shawn_Baugh_II/animal.py

```diff
@@ -0,0 +1,42 @@
+class Animal(object):
+    def __init__(self, name, health = 100):
+        self.name = name
+        self.health = health
+
+    def walk(self):
+        self.health = self.health - 1
+        return self
+    def run(self):
+        self.health = self.health - 5
+        return self
+    def displayHealth(self):
+        print self.name
+        print self.health
+        return self
+
+snake = Animal('snake').walk().walk().walk().run().run().displayHealth()
+
+class Dog(Animal):
+    def __init__(self, name, health = 150):
```

Comment:
Dog class doesn't need an __init__ method if you're not going to add extra properties to your Dog class.